PyDynamic/uncertainty/validate_FIRuncFilter.py

```diff
@@ -0,0 +1,62 @@
+import scipy.signal as scs
+import numpy as np
+import matplotlib.pyplot as plt
+
+import sys
+sys.path.append(".")
```

Comment:
Do we really want that in production code? I would rather not include such PATH hacks into the code base.

Reply:
:+1: